app/controllers/api/pages_controller.rb

```diff
@@ -58,6 +58,12 @@ def similar
     render :index, format: :json
   end
 
+  def total_donations
+    @page = Page.find(params[:page_id])
```

Comment:
We already have the `before_action :get_page` trigger.

Reply:
oh! ignore me then

Reply:
No by all means, it's not great - if you have an idea of how to make it work better with both parameter names (unless there are counter indications), let me know. I tried for a wee with the before_action filter before realizing that the parameter names were different and then I dropped it.